### ISSUE TITLE
add 'apply to be a member' page

### ISSUE THREAD

**Issue by derekeder** [chihacknight/chihacknight.org#164]
Now that we are accepting membership applications, we need to promote this on the website. Lets create a new page and embed the Google form on it: https://docs.google.com/forms/d/1bdLJ3BiwIhZbAEnwf0KsejTyaJydTzImvwjFSdMUU9c/edit?ts=5c0885c9

**Comment by easherma** [comment]
Where would we want this to live?
Three options stick out at me: 
* have a new totally separate page
* include it as part of about
* refactor 'donate' to be 'membership' and add the form there

**Comment by derekeder** [comment]
Lets have it on its own page. We will be promoting it a lot and the form itself has a few sections to it.

**Comment by derekeder** [comment]
Nav - not sure. We may need to reorganize the structure of it.

**Comment by easherma** [comment]
https://github.com/chihacknight/chihacknight.org/tree/164-member-page is a start

**Comment by derekeder** [comment]
I've setup our Paypal account to accept donations for membership. Here's the embed code:

```
<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
<input type="hidden" name="cmd" value="_s-xclick" />
<input type="hidden" name="hosted_button_id" value="822LXFNLVXQ5Y" />
<input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" />
<img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1" />
</form>
```

**Comment by derekeder** [comment]
closed via #208
